### ISSUE TITLE
Duplicate License Displayed on ~userInfo

### ISSUE THREAD

**Issue by adane** [Tizra/Tizra-Customer-Tracker#34]
Reported by Tim Huiner thuiner@g-w.com at G-W:
1.       I set up an account set with a license that has access for 365 days for our Adventures in Food and Nutrition title.
2.       In the account set, I added tags in the Matching all section.
3.       I created an user and added the same tags.
4.       I logged in as the user and I saw that I had a license for 365 days.
5.       When I tried to access the book, I was able to navigate as normal and access the book as expected.
6.       When I went back to My Bookshelf, however, I now saw 2 licenses, one with the correct Start and End Date and one with the original 365 day license.
7.       When I repeated this process with another user the next day, I was received the correct Start and End date, but I received the same display error.

In summary, everything is working well with the account set licenses, this is simply a display error that happens after the book is accessed.  Let me know if you have any other questions about this, thanks.

![G-W Screenshot](http://tizra.com/downloads/g-w-bugreport.png)

**Comment by daviddurand** [comment]
This is the result of the group license (which will let the user get _one_ individual license on attempted access), showing up in the display
even though the user's own personal license (which looks the same) has now been created.

So it's not a display bug so much as the implementation of the access control showing through. We'll have to change the display
to detect when group licenses are redundant in this way.

**Comment by daviddurand** [comment]
updates completed.
